.github/ISSUE_TEMPLATE/bug-issue-template.yml

```
@@ -0,0 +1,41 @@
+name: Bug report
+description: Bug report template.
```

P.S.: I recommend changing the filename to `bug_report.yml`.
```suggestion
description: "Create a report to help us improve"
```